models/netsuite.yml

```diff
@@ -48,7 +48,22 @@ models:
       either Retained Earnings or Net Income.  The Cumulative Translation
       Adjustment total, which in most cases does not have transactions
       associated with it, is calculated manually in the second part of the query.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - transaction_line_id
+            - transaction_id
+            - accounting_period_id
+            - account_name
```

Comment:
Is there a particular reason we chose account_name vs account_id?

Reply:
Same comment for the other additional tests.

Reply:
The only reason I can think of is if account names changed and we want to test on the name as opposed to the id. If that is what we have for Netsuite2 then I am happy keeping this as is!